js/movie_browser.js

```diff
@@ -1,2 +1,74 @@
-// API Docs at: 
-// http://www.omdbapi.com
\ No newline at end of file
+// API Docs at:
+// http://www.omdbapi.com
+$( document ).ready(function() {
+  console.log( "ready!" );
+});
+
+//on form submit this function runs two other functions
+function myFunction1(){
```

Comment:
Is there a particular reason you decided to set things up this way (including the `onsubmit` in HTML instead of setting up a JS event listener).